scriptworker/worker.py

```diff
@@ -62,7 +62,7 @@
         log.error("Hit ScriptWorkerException: {}".format(e))
     except Exception as e:
         log.exception("SCRIPTWORKER_UNEXPECTED_EXCEPTION task {}".format(e))
-        raise
+        status = STATUSES['internal-error']
```

Comment:
It's not really an internal error we're talking about here. It's a malformed upstream artifact.
Is there a reason we're fixing this here, rather than adding `TypeError` to https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/cot/verify.py#L2146 , which would raise a CoTError?

From my viewpoint:

- we set up a graph that never could have worked, due to task-graph.json being malformed
- we have a long term solution to creating task-graph.json consistently: using taskgraph everywhere
- this may be a good change, so we can take this as well. However, we also have the information to make a more surgical change that would give us a better exit code for #353 specifically.

Reply:
Ah, yes, I'll gladly add `TypeError` to `verify_chain_of_trust` - providing a more-relevant exit code for taskcluster is a big :+1: from me :smile:, good catch! I assumed that a schema would help here, but seeing as we have a top-level `except` block already, I'll hook onto that!